next.config.js

```diff
@@ -15,11 +15,15 @@ module.exports = withOptimizedImages({
     return config
   },
   async rewrites() {
-    return [
-      {
-        source: '/api/:path*',
-        destination: `${process.env.NEXT_PUBLIC_BACKEND_URL}/:path*`
-      }
-    ]
-  }
+    if (process.env.NODE_ENV === 'development') {
+      return [
+        {
+          source: '/api/:path*',
+          destination: `${process.env.NEXT_PUBLIC_BACKEND_URL}/:path*`,
+        },
+      ]
+    } else {
+      return []
```

Comment:
Why?

Reply:
We don't need proxying in prod because it's mapped by Traefik